pkg/cri/server/container_create_windows.go

```diff
@@ -129,3 +131,17 @@ func (c *criService) containerSpec(
 func (c *criService) containerSpecOpts(config *runtime.ContainerConfig, imageConfig *imagespec.ImageConfig) ([]oci.SpecOpts, error) {
 	return nil, nil
 }
+
+// snapshotterOpts returns any Windows specific snapshotter options for the r/w layer
+func snapshotterOpts(config *runtime.ContainerConfig) []snapshots.Opt {
```

Comment:
May want to take in the snapshotter name here and implement specific functionality based on what is passed. e.g. Some snapshotters may not support a label to set rootfs size.

Reply:
Good idea

Reply:
Added in a second parameter to take in the snapshotter name. Linux and other platforms are still bare, and Windows just checks "windows" for now as there's nothing in the container spec to take action with for LCOW at the moment (I suppose we could re-use the same Windows field but I'll worry about that whenever we think about LCOW for CRI potentially).